test/utils/image/manifest.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 const (
+	dockerRegistry  = "docker.io"
```

Comment:
nit: maybe rename to `dockerHubRegistry`? "docker registry" could mean many things...